app/import_data_v2.py

```diff
@@ -55,14 +55,27 @@ def insert_meta(input, type, company_id):
         # Iterating through the agencies in the json file
         for agency in data['agencies']:
 
+         # Validate region
+            if agency['region'].title() in ['Remote', 'Drenthe', 'Flevoland', 'Friesland', 'Gelderland', 'Groningen', 'Limburg', 'Noord-Brabant', 'Noord-Holland', 'Overijssel', 'Utrecht', 'Zuid-Holland', 'Zeeland']:
+                region = agency['region'].title()
+            else:
+                region = 'Remote'
```

Comment:
Maybe reject any data that does not pass the validation? 

You can even push these records to an array and write them to a file manually inspect them

Reply:
Or maybe we don't care 😂